src/plugins/application_config/README.md

```diff
@@ -0,0 +1,11 @@
+# applicationConfig
+
+A OpenSearch Dashboards plugin
+
+---
+
+## Development
```

Comment:
Can we add more details about this plugin? Here we can add instructions how to turn it on and off.

Reply:
Updated readme.